frontend/src/metabase/api/collection.ts

```diff
@@ -21,7 +33,49 @@ export const collectionApi = Api.injectEndpoints({
       providesTags: (response, error, { models }) =>
         provideCollectionItemListTags(response?.data ?? [], models),
     }),
+    getCollection: builder.query<Collection, CollectionRequest>({
+      query: ({ id, ...body }) => ({
+        method: "GET",
+        url: `/api/collection/${id}`,
+        body,
+      }),
```

Comment:
There's no `body` here. Only the `id`.
And since there's only `id`, the pattern we've used in all other API endpoints was to pass it directly as a sole argument, instead of in an object.

cc @iethree @sloansparger (for the future reference)